src/tests/end-to-end/common/page-controllers/page.ts

```diff
@@ -86,7 +86,11 @@ export class Page {
         return await this.screenshotOnError(async () => await this.underlyingPage.evaluate(fn, ...args));
     }
 
-    public async getMatchingElements<T>(selector: string, elementProperty: keyof Element): Promise<T[]> {
+    public async $(selector: string): Promise<any> {
+        return await this.screenshotOnError(async () => await this.underlyingPage.$(selector));
```

Comment:
This looks really similar to `getSelectorElement` below. Should we pick one/combine them?

Reply:
Done. Removed `$` and used `getSelectorElement` for my purpose. Good catch. Thanks.